data/sub

```diff
@@ -0,0 +1,14 @@
+[Script Info]
+; Script generated by bubblesub
+; https://github.com/bubblesub/bubblesub
```

Comment:
While it's a flattering advertisement, I'd avoid putting irrelevant project names in this repo

Reply:
Just created an ass file with `bubblesub` and placed there, no flattering advertisement.